android/app-example/src/test/java/com/badoo/ribs/example/rib/foo_bar/FooBarInteractorTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.badoo.ribs.example.rib.foo_bar
+
+import com.badoo.ribs.example.rib.foo_bar.feature.FooBarFeature
+import com.nhaarman.mockitokotlin2.mock
+import com.uber.rib.core.InteractorHelper
+import com.uber.rib.core.RibTestBasePlaceholder
+import io.reactivex.ObservableSource
+import io.reactivex.functions.Consumer
+
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+
+//@RunWith(RobolectricTestRunner::class)
+class FooBarInteractorTest : RibTestBasePlaceholder() {
```

Comment:
Do we want to have generated tests for our ribs for interactor + router?
All logic should be inside feature, probably we should update default generator to generate mock test file for feature.

Reply:
Well, not all logic, only state-related logic. I guess it does make sense to test if a certain thing is triggered in a child, then a certain configuration is pushed to the router, let's say.

Or in router it can make sense to test what ribs we definitely want to attach for a configuration.

But otherwise good point, we should have test for feature as well.